### PR TITLE
Addition of action-worker

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Included addition of action-worker

# What has changed
Basically a copy of action-scheduler but with name changes

# How to test?
Run in docker environment 

# Links
https://trello.com/c/QzSzSbP6

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[2021 PERFORMANCE: Action worker processor (8)](https://trello.com/c/QzSzSbP6/271-2021-performance-action-worker-processor-8)

**Comment by gavinleeedwards** [comment]
[2021 PERFORMANCE: Action worker processor part 2 (8)](https://trello.com/c/462ZhSzd/399-2021-performance-action-worker-processor-part-2-8)

File: rm-services.yml [hunk 1]
```diff
@@ -49,6 +49,28 @@ services:
...
+    container_name: actionworker
+    image: eu.gcr.io/census-rm-ci/rm/census-rm-action-worker
+    external_links:
+     - postgres
+     - rabbitmq
+    environment:
+     - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres:${POSTGRES_PORT}/${POSTGRES_HOST}?sslmode=disable
+     - SPRING_RABBITMQ_HOST=${RABBIT_HOST}
+     - SPRING_RABBITMQ_PORT=${RABBIT_PORT}
+     - EXCEPTIONMANAGER_CONNECTION_HOST=${EXCEPTIONMANAGER_HOST}
```
[the remark this answers is not in the document]
Comment (NickGrantONS) [reply rm-services.yml]: we might want to use it

**Comment by NickGrantONS** [comment]
> I don't think this should go in until we have image builds for the action worker.

Agree with @AdamHawtin but we now have the census-rm-deploy PR ready to go

File: rm-services.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdamHawtin) [reply rm-services.yml]: We can always add it back in? Seems better to keep it to just what it needs

**Pull request merged**